ui/dashboards/src/components/Variables/Variable.tsx

```diff
@@ -40,6 +40,7 @@ function ListVariable({ name }: TemplateVariableProps) {
   const { data: variablePlugin } = usePlugin('Variable', definition.spec.plugin.kind);
 
   const { setVariableValue, setVariableLoading, setVariableOptions } = useTemplateVariableActions();
+  const datasources = useLegacyDatasources();
```

Comment:
Will this eventually allow us to choose different datasources? It seems like it only supports `defaultDatasource` right now?

Reply:
Yup. I'm starting to work on "Datasources v2", so we should be able to plug that in here sometime in the next few weeks.